giskard_cicd/loaders/huggingface_loader.py

```diff
@@ -98,6 +101,8 @@ def load_giskard_model_dataset(
         # @TODO: currently for classification models only.
         logger.debug("Retrieving classification label mapping")
         if classification_label_mapping is None:
+            if hf_model is None:
+                hf_model = self.load_model(model_id)
```

Comment:
It seems good. My only concern is that: inference API is able to load more models than pipeline, this might fail. Because pipeline may require other deps.

I think I can resolve it by giving more deps to the venv.